### ISSUE TITLE
Datapoint.Timestamp in Local Time

### ISSUE THREAD

**Issue by WalkerCodeRanger** [aws/aws-sdk-net#167]
The [docs for `Datapoint.Timestamp`](http://docs.aws.amazon.com/sdkfornet/latest/apidocs/items/PCloudWatchDatapoint_Timestamp_NET4_5.html) state "The time stamp used for the datapoint. Amazon CloudWatch uses Coordinated Universal Time (UTC) when returning time stamps, which do not accommodate seasonal adjustments such as daylight savings time. For more information, see Time stamps in the Amazon CloudWatch Developer Guide."  However, the datapoints returned from `GetMetricStaticsAsync` using SDK 2.3.23.0 are in local time and have `Kind` equal to local.  I find this behavior very surprising, unexpected and confusing.  Especially considering that you must specify the `StartTime` and `EndTime` in UTC when using `GetMetricStaticsAsync` or you will not get the correct data.

In short, you don't seem to be handling `DateTime.Kind` correctly either on the way in or the way out.  I would expect that values taken into the API would be converted to UTC or rejected if they are in local time and values coming out would be in Utc with kind Utc.

**Comment by PavelSafronov** [comment]
These are entirely valid points, but this is a very large breaking change that we cannot make at this point.

**Comment by johnsimons** [comment]
We have just run into this issue, sad that it cannot be fixed.
And the doco in https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_Datapoint.html does not state what datetime kind the timestamp is in.

**Comment by ramonsmits** [comment]
This seems fixed in `AWSSDK.CloudWatch` Version 4.0.6.1, Kind is `Utc` and timestamps in the datapoints also align with my numbers tested for my GMT+1 timezone Europe/Amsterdam
